### PR TITLE
ETS-2026 removes space between header and content

### PR DESCRIPTION
**Pull request by BirgitPohl**
## 0.38.0
- Removes the created space under the header
- Cleaned up code and made it more stable and readable

### CODE REVIEW

**Comment by BirgitPohl** [comment]
PR will be squashed

File: source/scss/layout/_header.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@ body > header {
   display: block;
   margin: 0 auto;
   min-width: 320px;
+  position: sticky;
```
Comment (filipe-esposito-fti) [inline source/scss/layout/_header.scss:9]: I believe there isn't still much support for this feature... Please see this: https://caniuse.com/#feat=css-sticky

Comment (filipe-esposito-fti) [reply source/scss/layout/_header.scss:9]: Or you added this piece of code here to help with that? https://github.com/tourstream/fti-portal-styleguide/pull/116/files#diff-6e172d48821ec5d2e2138edc9f0a9bd6R51

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## 0.38.0
+- Removes the created space under the header
```
Comment (filipe-esposito-fti) [inline CHANGELOG.md]: I think _space created_ sounds better :D

File: source/scss/layout/_header.scss (hunk 1 above)
Comment (BirgitPohl) [reply source/scss/layout/_header.scss:9]: Exactement. 😉

**Review by filipe-esposito-fti — APPROVED** [review]

**Pull request merged**